test/test_utils.py

```diff
@@ -126,6 +126,31 @@ def single_build(request, recipes_fixture):
     yield recipes_fixture.pkgs['one']
     for pkg in recipes_fixture.pkgs['one']:
         ensure_missing(pkg)
+        
+
+@pytest.fixture(scope='module', ids=IDS)
```

Comment:
@daler I have no idea how to see if this actually was tested and what its output was.